app-editors/juffed/juffed-0.10_p20200103.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+COMMIT=74ab7236a30be17351edc5b83eb3579affd96913
+inherit cmake xdg
+
+DESCRIPTION="QScintilla-based tabbed text editor with syntax highlighting"
+HOMEPAGE="http://juffed.com/en/"
+SRC_URI="https://github.com/Mezomish/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/Mezomish-${PN}-74ab723"
```

Comment:
How about replacing the inline short commit id with cutting it out of `${COMMIT}`?